spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonAutoConfiguration.java

```diff
@@ -268,15 +269,15 @@ public SpringAggregateSnapshotter aggregateSnapshotter(Configuration configurati
                                                            ParameterResolverFactory parameterResolverFactory,
                                                            EventStore eventStore,
                                                            TransactionManager transactionManager,
-                                                           SpanFactory spanFactory) {
+                                                           SnapshotterSpanFactory spanFactory) {
```

Comment:
This indentation seems off to me.

Reply:
Hmm seems right in my IDE
![image](https://github.com/AxonFramework/AxonFramework/assets/3278878/0b538012-9c1c-457d-8b3f-bbfc51d334b8)

Reply:
:thinking: I must say, I fully agree. Not sure how this comment got here...my apologies!